tests/test-console-time-manual.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2017, Intel Corporation.
+
+// test console.time() and console.timeEnd()
+
+var performance = require("performance");
+
+var before = performance.now();
+
+console.log("test: 20 timers run at the same time");
+for (var i = 1; i < 21; i++) {
+    console.time("Timer" + i);
+}
+
+setTimeout(function() {
+    for (var j = 1; j < 20; j++) {
+        console.timeEnd("Timer" + j);
+
+        var after = performance.now();
+        var diff = after - before;
```

Comment:
Add a "| 0" to the end of this line to round it off to the nearest ms which will make the output more readable.